Addition of landing page text (#118)
Sydney-Informatics-Hub/Weed-ID-Interchange: README.md

# Welcome to the Weeds Imaging Database (WeedID). 

> WeedID Is an open source, searchable, weeds image platform designed to facilitate the research and development of machine learning algorithms for weed recognition in cropping systems.

Reviewer: ```suggestion
WeedID is an open source, searchable, weeds image platform designed to facilitate the research and development of machine learning algorithms for weed recognition in cropping systems.
```

Camera-based weed recognition using deep learning algorithms has emerged as a frontrunner for in-crop site-specific control with an improved ability to handle variation.
Training and development of algorithms require significant quantities of high-quality, annotated image data.
WeedID is addressing this challenge by enabling the easy access and contribution of weed image data on an open source platform with search, dynamic filter and preview functions for custom dataset download capability. 

# Data supported 

Annotation data is stored in the Weed-COCO format, an extension of MS COCO for weed recognition in grain crops. Weed-COCO incorporates additional whole-dataset contextual information on: 

- Crop type 

- Crop growth stage (text and BBCH) 

- Soil colour 

- Surface coverage 

- Weather description 

- Location 

- Camera metadata (camera model, collection height, angle, lens, focal length, field of view) 

- Lighting  

The format may also be applicable to related agricultural purposes.

> As with MS COCO, the format supports classification, bounding box and segmentation labels.

Reviewer: ```suggestion
As with MS COCO, the format supports classification, bounding box and segmentation labels indicating the presence of a specific or unknown species of weed.
```

Reporting these details will help ensure consistency in published datasets for ease of comparison and use in further research and development.

![WeedID Data Flow](weedID-data-flow-diagram.png)

# Acknowledgements 

This project has been funded by the Grains Research and Development Corporation.

> The platform was developed in a collaborative effort between University of Sydney research groups, the Sydney Informatics Hub, Australian Centre for Field Robotics and the Precision Weed Control Group. 

Reviewer: Should this be indicating that the research groups are as follows, i.e.

```
The platform was developed in a collaborative effort between University of Sydney research groups**:** the Sydney Informatics Hub, Australian Centre for Field Robotics and the Precision Weed Control Group. 
```

Technically we are not a research group, but a core research facility, and our CRF overlords are particular about us being acknowledged as such.

Maybe:
```suggestion
The platform was developed by the Sydney Informatics Hub, a core research facility of the University of Sydney, as part of a research collaboration with the Australian Centre for Field Robotics and the Precision Weed Control Group at the University of Sydney.
```

# Citation Guidelines 

## General 

If you found WeedID useful in your research, please cite the database as: 

> INSERT_CITATION 

Reviewer: ```suggestion
CITATION TBA by mid 2021
```
